Source/WebKit/NetworkProcess/cache/NetworkCacheStorage.cpp

```diff
@@ -969,7 +969,7 @@ void Storage::store(const Record& record, MappedBodyHandler&& mappedBodyHandler,
     m_writeOperationDispatchTimer.startOneShot(m_initialWriteDelay);
 }
 
-void Storage::traverse(const String& type, OptionSet<TraverseFlag> flags, TraverseHandler&& traverseHandler)
+void Storage::traverseInternal(const String& rootPath, const String& type, OptionSet<TraverseFlag> flags, TraverseHandler&& traverseHandler)
```

Comment:
I don't like calling things FooInternal. Can this say something about what it actually does like `traverseWithinRootPath`?

Reply:
Sounds good.